.github/workflows/status-check-outlets.yml

```diff
@@ -1,6 +1,8 @@
 name: Status check on supported outlets
 
 on:
+  push:
```

Comment:
You probably added this temporarily to get the CI to run automatically? That's fair, but it shouldn't be merged into `main`. I'm not very familiar with Github's CI, but there might be another way to set this up (or you could run it manually).

Reply:
Yeah, I should probably have made another branch to add that too.